packages/framework/src/Command/ComposerScriptHandler.php

```diff
@@ -17,6 +17,7 @@ public static function postInstall(Event $event)
     {
         static::executeCommands($event, [
             'shopsys:domains-urls:configure',
+            'ckeditor:install --clear=skip --release=full --tag=4.5.11',
```

Comment:
I believe this is covered by the two points in commit message
```
- due to license reasons FOSCKEditor downloads frontend scripts via command ckeditor:install
- version of frontend part was locked into 4.5.11 to match previous skin and also the skin of fm_elfinder
```

However, I would appreciate a little bit more information. Why it is not implemented as a phing target? Does it need to be locked? Why do we wanna keep the previous skins? :grin: 
Maybe it would make sense to extract this change into a separate commit? :thinking:

Reply:
splited into 2 commits with link onto symfony documentation about this package.